db/dcrsqlite/apisource.go

```diff
@@ -1262,3 +1262,19 @@ func (db *wiredDB) CountUnconfirmedTransactions(address string, maxUnconfirmedPo
 	}
 	return
 }
+
+//HeightOfSpendingTxForVotes gives the block height of the transaction id specified
+func (db *wiredDB) HeightOfSpendingTxForVotes(txid string) (height int64) {
```

Comment:
This should be renamed as it has nothing to do with votes or spending txns.  `TxHeight` would be more appropriate.